packages/integration/ops/config/router/config.json

```diff
@@ -5,13 +5,17 @@
       "providers": ["http://172.17.0.1:8545"],
       "confirmations": 1,
       "subgraph": "http://172.17.0.1:8010/subgraphs/name/connext/nxtp",
-      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0"
+      "transactionManagerAddress": "0x8CdaF0CD259887258Bc13a92C0a6dA92698644C0",
+      "feePercentage": 0.05,
```

Comment:
This 0.05% fee should not be allowed to be configured by users. The idea is this is a flat fee across the network.

Reply:
Router should be able to change the fee they want charge.